measurement/util.py

```diff
@@ -143,6 +144,13 @@ def parse_insight_opts():
     parser.add_argument("--vmtouch-target", action="store", default=None,
                         help="File or dir to be diagnosed.")
 
+    parser.add_argument("-b", "--blktrace", action="store_true", default=False,
```

Comment:
I think it's not necessary to use `-b` for this argument.

Reply:
as well as `-f` argument for ftrace